docs/getting-started-guides/kubeadm.md

```diff
@@ -160,15 +160,14 @@ Before you can deploy applications to it, you need to install a pod network.
 You must install a pod network add-on so that your pods can communicate with each other when they are on different hosts.
 **It is necessary to do this before you try to deploy any applications to your cluster.**
 
-Several projects provide Kubernetes pod networks.
-You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
+Several projects are available that enable Kubernetes pod networks including Calico, Romana and Weave. Some may also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
```

Comment:
In general, I'd personally like to move the opposite direction - that is, making _fewer_ callouts to specific networking solutions in the generic k8s docs, instead directing users to a single, centralized list (e.g networking.md)

I understand this isn't the case right now and there are a bunch of places that make specific call outs to flannel, weave, calico, etc, but as more solutions start to join the k8s ecosystem I fear this has a real potential to get unmaintainable, fast.  WDPT?